_haikus/haiku_you.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Haiku You
+author: HereToLearn101
+---
+
+Haiku rhymes with you
```

Comment:
Please add a `<br>` tag or two spaces `  ` following the lines so that you can force a line break.

Reply:
ok I'll put those '<br>' tags in. Thank you